material/problems/VectorBasics/Spool/Spool.tex

```diff
@@ -13,13 +13,47 @@
 %%%
 %%%
 %%%
-The figure below shows a rigid spool \basis{B} in contact with the ground \basis{N}. Denote the spool's inner radius as $r$ and outer radius as $R$.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+The figure below shows a rigid spool \basis{B} in contact with the ground
+\basis{N}. Denote the spool's inner radius as $r$ and outer radius as $R$.
 The point of the spool in contact with the ground is $B_N$.
-In completing this problem, please introduce appropriate unit vectors, identifiers (labels), and anything else you need to help you.
+In completing this problem, please introduce appropriate unit vectors,
+identifiers (labels), and anything else you need to help you.
 \begin{enumerate}
-\item Calculate \moment{\bvec{F}_T/B_N}, the moment due to force \force{T} about $B_N$, in terms of $r$, $R$, $\theta$, and any other quantities necessary.
-\item Compute the numerical value of \moment{\bvec{F}_T/B_N} when $F_T = 10$ N, $r = 1$ m, $R = 2$ m, and $\theta = \degrees{60}$.
+  \item Calculate \moment{\bvec{F}_T/B_N}, the moment due to force \force{T}
+    about $B_N$, in terms of $r$, $R$, $\theta$, and any other quantities necessary.
+  \item Compute the numerical value of \moment{\bvec{F}_T/B_N} when $F_T = 10$ N,
+    $r = 1$ m, $R = 2$ m, and $\theta = \degrees{60}$.
 \end{enumerate}
-\vspace{4.0pc}
-\includegraphics[height=5.5cm]{spool.png}
-\isolatedBuildFooter
\ No newline at end of file
+\begin{minipage}[t]{0.44\linewidth}
+  \minipageTopAnchor
+  \flushleft
+  \vspace*{-3cm}
+  \begin{center}
+    \includegraphicsAB[width=1\linewidth]
+      {spool-solution.png}
+      {spool.png}
+  \end{center}
+\end{minipage}
+\Solution{}{0.53\linewidth}{
+  \begin{minipage}[t]{0.99\linewidth}
+    \begin{tabular}{lcl}
+      \moment{\bvec{F}_T/B_N} & $\stackrel{\triangleSymbol}{\equals}$ & \posvec{B_N}{B_I}~\mult $~\bvec{F}_T$
+      \\[1.0pc]
+      &=&(R~\uvecy{n}-r~\uvecy{b})~\mult ~($F_T$~\uvecx{b})
+      \\[1.0pc]
+      &=&$F_T$R~\uvecy{n}~\mult~\uvecx{b}\minus$F_T$r~\uvecy{b}~\mult~\uvecx{b}
+      \\[1.0pc]
+      &=&$F_T$R(\minus~\uvecz{n})$~\sin$(90\minus $\theta$) \minus $F_T$r(\minus~\uvecz{n})
```

Comment:
For all these lines, everything in the right column of the table should be inside `$...$`, otherwise the formatting doesn't look right.